joomla/joomla-cms
# [4.0] Removing extra Expired Cache view and menu item

This moves the "Delete Expired Cache" feature from its own (empty) view into the general "Clear Cache" view.

### What does this do?
This 
- removes the submenu at the side
- removes the purge view
- adds the button to delete expired cache into the general "clear cache" view
- removes the menu item from the system view

Please see that I removed the check for the token. As far as I can see, that check is not necessary. My plan is also to add this as a quicktask to the menu item of the "Clear Cache" menu item of the system view in the new backend template. A quicktask allows you to define a link with an icon that is attached at the end of a menu item. Clicking on that directly brings you to a new article screen or in this case clears the expired cache, etc. In any case, I would need a green light from @SniperSister or someone else from the security team for this change.

### How to test?
1. Go to the "System" menu item in the backend and see the menu item to purge expired cache. Click on that menu item and see that you get a mostly empty screen with a button at the top and a submenu with 3 entries on the left. Also a message is displayed about performance considerations.
2. Apply PR
3. See that the menu item is gone. Click on "Clear Cache"
4. See that the submenu is gone and an additional button has appeared to clear the expired cache.
5. Click on that button and get the previously displayed message about the performance considerations. If aborted, nothing happens, if agreed, the cache is cleared from expired entries.
